syntax = "proto3";

package inference;

message InferenceRequest {
  bytes tensor_data = 1;
}

message InferenceResponse {
  bytes tensor_data = 1;
}

service InferenceService {
  rpc RunInference(InferenceRequest) returns (InferenceResponse);
}